src/clusterfuzz/_internal/tests/core/bot/fuzzers/googlefuzztest/googlefuzztest_engine_test.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for googlefuzztest engine."""
+# pylint: disable=unused-argument
+import os
+import unittest
+
+from clusterfuzz._internal.bot.fuzzers import engine_common
+from clusterfuzz._internal.bot.fuzzers.googlefuzztest import engine
+from clusterfuzz._internal.tests.test_libs import test_utils
+
+TEST_PATH = os.path.abspath(os.path.dirname(__file__))
+DATA_DIR = os.path.join(TEST_PATH, 'test_data')
+TEMP_DIR = os.path.join(TEST_PATH, 'temp')
+GOOGLEFUZZTEST_ENGINE_TEST_SUFFIX = 'googlefuzztest_engine_test'
+
+
+#Forcing to skip this in local unit test runs, as the generated binary only works in CI
+@test_utils.integration
```

Comment:
Unfortunately I don't think this is sufficient. We can't have a test that fails locally and it's normal to run intergration tests locally. I think we probably should just drop the tests that use the googlefuzztest binary for now and possibly complain to the googlefuzztest team to see if they can change this situation.

Reply:
Will file a ticket to the fuzztest folks describing how to reproduce the issue. Will tackle this again once we figure out why stuff is breaking

Reply:
I'm not sure they will be able to help.